README.md

```diff
@@ -86,12 +86,12 @@ $ cd watson-vehicle-damage-analyzer
 
   * [**Watson Visual Recognition**](https://console-regional.ng.bluemix.net/catalog/services/visual-recognition)
 
-## 3. Add Visual Recoginition API key to .env file
+## 3. Add Visual Recognition API key to .env file
 
 When you create the Visual Recognition service, you will need the API key
 ![](doc/source/images/apiKey.png)
 
-Move the watson-vehicle-damage-analyzer/server/env.example file to ``/.env`` and populate the API key:
+Move the `watson-vehicle-damage-analyzer/server/env.example file to ``/.env`` and populate the API key:
```

Comment:
Missing ending backpack. Should be:
Move the `watson-vehicle-damage-analyzer/server/env.example` file to ``/.env`` and populate the API key: